pynxtools/dataconverter/readers/em/concepts/concept_mapper.py

```diff
@@ -0,0 +1,116 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Utilities for working with NeXus concepts encoded as Python dicts in the concepts dir."""
+
+# pylint: disable=no-member
+
+import pytz
+
+from datetime import datetime
+
+
+def load_from_modifier(terms, fd_dct):
+    """Implement modifier which reads values of different type from fd_dct."""
+    if isinstance(terms, str):
+        if terms in fd_dct.keys():
+            return fd_dct[terms]
+    if all(isinstance(entry, str) for entry in terms) is True:
+        if isinstance(terms, list):
+            lst = []
+            for entry in terms:
+                lst.append(fd_dct[entry])
+            return lst
+    return None
+
+
+def convert_iso8601_modifier(terms, dct: dict):
+    """Implement modifier which transforms nionswift time stamps to proper UTC ISO8601."""
+    if terms is not None:
+        if isinstance(terms, str):
+            if terms in dct.keys():
+                return None
```

Comment:
Why is this needed?

Reply:
need to check, have noted this down but resolvable